tools/create/src/templates/component/package.json.ts

```diff
@@ -25,8 +25,8 @@ export const pkgJson = ({
     "access": "public"
   },
   "dependencies": {
-    "@leafygreen-ui/emotion": "^4.0.4",
-    "@leafygreen-ui/lib": "^10.4.0"
+    "@leafygreen-ui/emotion": "^4.0.7",
+    "@leafygreen-ui/lib": "^12.0.0"
```

Comment:
can we automate this somehow @TheSonOfThomp

Reply:
Yes, just need to write the script. It'll involve pinging npm when running `lg create`

Reply:
https://jira.mongodb.org/browse/LG-3663